pkg/kubectl/genericclioptions/resource/builder.go

```diff
@@ -1141,3 +1141,13 @@ func MultipleTypesRequested(args []string) bool {
 	}
 	return rKinds.Len() > 1
 }
+
+func AnyCategoryRequested(categoryExpander restmapper.CategoryExpander, args []string) bool {
+	for _, arg := range args {
+		_, ok := categoryExpander.Expand(arg)
```

Comment:
> btw should we print the kinds on e.g. "kubectl get ingress,[catetory]". If so, we should looping over the args to check if any element is a category instead of args[0] the very first one. WDYT?

We can always specify one category at a time. :)

So the category will always be `args[0]` and `len(args)==1`.